source/cellular_pktio.c

```diff
@@ -877,14 +874,12 @@ static bool _handleDataResult( CellularContext_t * pContext,
 static bool _getNextLine( CellularContext_t * pContext,
                           char ** ppLine,
                           uint32_t * pBytesRead,
+                          uint32_t currentLineLength,
                           CellularPktStatus_t pktStatus )
 {
-    uint32_t stringLength = 0;
+    uint32_t stringLength = currentLineLength;
```

Comment:
Why don't you use `currentLineLength` directly?

Reply:
Update in the PR.